### ISSUE TITLE
semantic html

### ISSUE THREAD

**Issue by wildskyf** [RexYuan/courseNTNU#9]
There are something wrong with html in template.
`<h1>`, `<h2>`, `<h3>` should be used in heading. I will solve the problem.

**Comment by RexYuan** [comment]
@wi1d5ky 你最近能夠修這個嗎？
可以看看 [Bootstrap 的定義](https://github.com/twbs/bootstrap/blob/master/less/type.less)

``` css
h1, h2, h3, h4, h5, h6,
.h1, .h2, .h3, .h4, .h5, .h6 {
  font-family: @headings-font-family;
  font-weight: @headings-font-weight;
  line-height: @headings-line-height;
  color: @headings-color;

  small,
  .small {
    font-weight: normal;
    line-height: 1;
    color: @headings-small-color;
  }
}

h1, .h1,
h2, .h2,
h3, .h3 {
  margin-top: @line-height-computed;
  margin-bottom: (@line-height-computed / 2);

  small,
  .small {
    font-size: 65%;
  }
}
h4, .h4,
h5, .h5,
h6, .h6 {
  margin-top: (@line-height-computed / 2);
  margin-bottom: (@line-height-computed / 2);

  small,
  .small {
    font-size: 75%;
  }
}

h1, .h1 { font-size: @font-size-h1; }
h2, .h2 { font-size: @font-size-h2; }
h3, .h3 { font-size: @font-size-h3; }
h4, .h4 { font-size: @font-size-h4; }
h5, .h5 { font-size: @font-size-h5; }
h6, .h6 { font-size: @font-size-h6; }
```

但我覺得你覺得怎麼改比較好看比較重要

**Comment by RexYuan** [comment]
I'm closing this issue because this will be resolved at #26
